release-notes/v1.3.0.md

```diff
@@ -0,0 +1,20 @@
+# HPE CSI Driver for Kubernetes 1.3.0 Release Notes
+
+## HPE CSI Driver for Kubernetes 1.3.0
+
+| **Version:** |1.3.0|
+|--------------|-----|
+| **Revision:** | Tuesday Sep 08, 2020 |
+
+## Resolved Issues
+
+The following table lists the resolved issues for HPE CSI Driver for Kubernetes v1.3.0.
+
+## Known Issues
+
+The following table lists the known issues for HPE CSI Driver for Kubernetes v1.3.0. Please note that, `Known Issues` from previous releases are still applicable with suggested workarounds, if they are not part of `Resolved Issues` above.
+
+|ID|Component |Title|Description|Workaround|
+|--|---------|-----|-----------|----------|
+|CON-1107|csi.k8s|Restoring NFS access on extended outage with backends| If PVC's are created with NFS access using `HPE CSI Driver`, then a `NFS Server Provisioner` instance will be created for each PVC. Since these servers are backed by underlying FC/iSCSI disks. If there is an extended outage (> 150s) with the backend, then NFS servers will get I/O failures and cannot service NFS shares anymore. Since all filehandles are stale, NFS servers does not automatically recover if access is restored later on and application pods can be hung due to hard mounts|In these cases we recommend to restart affected NFS Server Provisioner pods or the node to restore iSCSI/FC disk access again and serve NFS mounts|
```

Comment:
nit: PVCs

incomplete sentence: Since these servers are backed by underlying FC/iSCSI disks. 

NFS servers do not